taf/developer_tool.py

```diff
@@ -969,6 +1063,11 @@ def _write_targets_metadata(taf_repo, targets_roles, keystore, roles_key_infos,
             update_method = taf_repo.roles_keystore_update_method(role_name)
             update_method(keystore_keys, write=False)
 
+
+def _write_targets_metadata(taf_repo, targets_roles, keystore, roles_key_infos, scheme):
```

Comment:
Name of the function is `_write_targets_metadata`, but it updates "snapshot", "timestamp" as well. As in previous comment, I am not sure if those should always be signed. Does this affect a build process where just target metadata should be generated?

Reply:
I think that it's fine because we always want to sign snapshot and timestamp in this case. `developer_tool` is not used by our build process